modules/ROOT/pages/prepare-to-install-service-mesh.adoc

```diff
@@ -23,6 +23,10 @@ Tip: To verify installed version, use the command: > helm version.
 For more information about how to install Helm, see the Helm installation documentation. https://helm.sh/docs/install/.
 ** Kubectl version 1.12 or later (compatible with the cluster) is installed on your local machine:
 For more information about how to install Kubectl, see https://kubernetes.io/docs/tasks/tools/install-kubectl/#before-you-begin 
+** One of the following environments on which to run Kubernetes:
```

Comment:
Please change this to 
These are supported Kubernetes providers

Reply:
Means the same that I have there. If I change it to your suggestion, the format will not be parallel to the others. So can't change.